src/pages/questionPage.js

```diff
@@ -32,7 +33,15 @@ export const initQuestionPage = () => {
 };
 
 const nextQuestion = () => {
-  quizData.currentQuestionIndex = quizData.currentQuestionIndex + 1;
-
+  if (quizData.currentQuestionAnswer===null){
+    const userInterface = document.getElementById(USER_INTERFACE_ID);
+    const alertElement = createAlertElement();
+    userInterface.appendChild(alertElement);
+  
+  }
+  else if (questions[currentQuestionIndex].correct === quizData.currentQuestionAnswer){
```

Comment:
I'm seeing some unformatted code, can you format this using Prettier? Maybe you also want to enable 'Format on save' on your VSCode settings